content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -40,8 +40,9 @@ feature gates. The `RuntimeClass` feature gate must be enabled on apiservers _an
 
 #### 1. Configure the CRI implementation on nodes
 
-The configurations available through RuntimeClass are CRI implementation dependent. See the
-corresponding documentation ([below](#cri-documentaiton)) for your CRI implementation for how to configure.
+The configurations available through RuntimeClass are Container Runtime Interface (CRI)
+implementation dependent. See the corresponding documentation ([below](#cri-documentation)) for your
+CRI implementation for how to configure.
```

Comment:
Looks good, 👍 
nit: the `below` link should reference (#cri-configuration) on the page, not #cri-documentation?